data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,734 @@
+		not "clobbered"
+	`	The soldiers give you a moment to dress yourself and gather your things back up, though the surly looks you're fixed with make it clear they're keen to see you go.`
+	label complete
+	action
+		payment -1000000
+	`	The transaction is complete. You've spent a million credits out of your own pocket for something that needs to be delivered to a mysterious mercenary you met in a stinking bar. This is definitely the path of wisdom that you walk. Thankfully, no further obstacles ambush you as you return to your ship.`
+		accept
+	label retreat
+	`	You make your apologies - wrong hangar - and beat a hasty retreat. The door seals behind you. On the way to your ship you pass the janitor once more, his ear firmly pressed to the handle of his mop. Upon seeing you he becomes visibly startled and drops his tools all over the floor. What a strange fellow.`
+		decline
+	
+
+
+mission "Pirate Intro: Domination"
+	landing
+	name "Dominate a planet"
+	description `Dominate Darkstone.`
+	destination "Greenrock"
+	waypoint "Han"
+	to offer
+		has "I Want To Be A Pirate: active"
+	to fail
+		has "I Want To Be A Pirate: done"
+	on enter "Han"
+		dialog `To dominate a planet, target the planet by clicking it or attempting to land, then hail. Select "Demand Tribute", and if your combat rating is high enough to be a threat, ships will take off to face you. Defeat every ship sent up, then hail the planet and "Demand Tribute" again to claim your prize!`
+	to complete
+		or
+			has "tribute: Darkstone"
+			has "hero of Darkstone"
+
+
+
+mission "Pirate Intro: Domination Check-in"
+	clearance
+	invisible
+	landing
+	source "Darkstone"
+	to complete
+		has "event: slaver encounter"
+	to fail
+		not "Pirate Intro: Domination: active"
+	to offer
+		has "Pirate Intro: Domination: active"
+		has "tribute: Darkstone"
+	on offer
+		event "slaver encounter" 2
+		conversation
+			`Apparently the mess you made in orbit was visible even from the ground, as when you come in to land there's a delegation of frankly depressed-looking colonists waiting to greet you. The only member of the party who looks even vaguely chipper is a goat that they're holding on the end of a rope, who's otherwise merrily grazing on whatever foliage clings to the edge of the packed-dirt landing pad.`
+			choice
+				`	(Wait for the colonists to speak first.)`
+					goto silence
+				`	"You lot know how it works. Old master's dead, that makes me the new one. Pay up and I'll be off."`
+					goto tribute
+				`	"Kneel before me, peasants!"`
+					goto silence
+				`	"Hello! I'm fairly new to this whole 'conquering the galaxy' thing, but I think that beating that crew of scoundrels like I did makes me the new boss of this place, right?"`
+			`	The colonists exchange looks. They seem far too lost in their own misery to even try dealing with your obnoxiously chipper demeanor, and you're simply given a chorus of shrugs and murmured 'Yah, sure, whynot' responses.`
+			`	"Whatever you say, chief."`
```

Comment:
```suggestion
			`	The colonists exchange looks. They seem far too lost in their own misery to even try dealing with your obnoxiously chipper demeanor, and you're simply given a chorus of shrugs and murmured "Yah, sure, whynot" responses. "Whatever you say, chief," one mutters.`
```
Having that on a line by itself made it confusing who exactly was speaking to me.